compiler/rustc_privacy/messages.ftl

```diff
@@ -8,6 +8,8 @@ privacy_from_private_dep_in_public_interface =
 privacy_in_public_interface = {$vis_descr} {$kind} `{$descr}` in public interface
     .label = can't leak {$vis_descr} {$kind}
     .visibility_label = `{$descr}` declared as {$vis_descr}
+    .help = - Either remove the `pub` from the function
```

Comment:
This says "from the function" -- but this isn't always fired on a function, is it?

Reply:
Thanks for reviewing my changes. I am not sure if it's always fired on a function, but you are most likely correct. In that case, what would be a better suggestion?